baracoda/formats/interfaces.py

```diff
@@ -0,0 +1,11 @@
+from baracoda.exceptions import UnsupportedChildrenCreation
+
+
+class FormatterInterface:
```

Comment:
Maybe a comment, like in the Readme, saying this is the interface to implement when adding new formatters

Reply:
Ok